frontend/src/metabase/visualizations/components/skeletons/util/display-type.ts

```diff
@@ -0,0 +1,20 @@
+export const chartSkeletonDisplayTypes = [
```

Comment:
I would't define viz types like that. They can be extended and can be something outside of this list. That's why there is `CardDisplayType` and it's a `string`. Could you use that type instead of this one?

Reply:
Yep, that makes sense. I also use the `TableSkeleton` as the default skeleton component as the default in the `switch` statement if a skeleton isn't found:

https://github.com/metabase/metabase/blob/38c15a7bba73f1ba1e33f35ac7511c497af851b1/frontend/src/metabase/visualizations/components/skeletons/ChartSkeleton/ChartSkeleton.tsx

Reply:
Also added a test for the display type  `"a random display type"` just to make sure that strings outside of the ones in the switch statement still render